src/ui/components/VPNCheckBoxAlert.qml

```diff
@@ -8,17 +8,18 @@ import Mozilla.VPN 1.0
 import "../themes/themes.js" as Theme
 
 RowLayout {
-    id: turnVPNOffAlert
     property var leftMargin:  56
+    property var alertColor: Theme.checkBoxWarning
+    property var errorMessage: "..."
```

Comment:
This is not exposed, right?

Reply:
It's not exposed currently but I just pushed up something to remove the placeholder text so it doesn't get accidentally exposed in the future.